Source/Marvirc/Bin/Join.php

```diff
@@ -195,6 +195,21 @@ public function main ( ) {
             return;
         });
 
+        // Message.
+        $client->on('message', function ( $bucket ) use ( $self ) {
+
+            $data   = $bucket->getData();
+            $answer = null;
+
+            if(0 < preg_match('/!(?=\b)/', $data['message']))
```

Comment:
> preg_match() returns 1 if the pattern matches given subject, 0 if it does not, or FALSE if an error occurred.

So what about if `false` happen ? And in this case, I think it should be better to use `1 ===` 

What do you think ?